server/app/filters/SettingsFilter.java

```diff
@@ -26,11 +27,14 @@ public SettingsFilter(Provider<SettingsService> settingsService, Materializer ma
     this.materializer = checkNotNull(materializer);
   }
 
+  private static final ImmutableList<String> EXCLUDED_PATHS =
+      ImmutableList.of("/assets/", "/favicon", "/playIndex");
```

Comment:
**thought:** Will be fun to see this decrease the number of DB calls in load testing :)

Reply:
Hope so!